README.md

```diff
@@ -1,7 +1,9 @@
 #<a name="top"></a>Orion Context Broker
 
-[![license badge](https://img.shields.io/badge/license-AGPL-blue.svg)](https://opensource.org/licenses/AGPL-3.0)
-[![Documentation Status](https://readthedocs.org/projects/fiware-orion/badge/?version=latest)](http://fiware-orion.readthedocs.org/en/latest/?badge=latest)
+[![License badge](https://img.shields.io/badge/license-AGPL-blue.svg)](https://opensource.org/licenses/AGPL-3.0)
```

Comment:
Note how this badge is rendered right now:

![2016-02-26 14_46_18-fiware-orion_readme md at 0 27 0 telefonicaid_fiware-orion](https://cloud.githubusercontent.com/assets/1534240/13353921/e72da986-dc97-11e5-9a81-e7f9a368a933.png)

Thus, the release seletec is "0.27.0" but the badge shows "doc | latest" instead of "doc | 0.27.0".

Could that be fixed with a badge line in the README.md that automatically renders the right documentation version? I.e. something like that?

```
[![Documentation badge](https://readthedocs.org/projects/fiware-orion/badge/?version=$current_tag)](http://fiware-orion.readthedocs.org/en/latest/?badge=$current_tag)
```

(This is just a nice to have; if it cannot be solved with a "static" line in README.md, we can implemented the adjustment in `release.sh` so the README.md is edited each time we step version).